src/test/java/com/browserstack/test/suites/product/FilterTest.java

```diff
@@ -0,0 +1,47 @@
+package com.browserstack.test.suites.product;
+
+import com.browserstack.test.suites.TestBase;
+import com.browserstack.test.utils.CsvUtil;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FilterTest extends TestBase {
+
+    @Test
+    public void filterLowestToHighestTest() {
+        Select sortSelect = new Select(driver.get().findElement(By.cssSelector(".sort select")));
+        sortSelect.selectByValue("lowestprice");
+
+        wait.until(ExpectedConditions.textToBePresentInElementLocated(By.xpath("//*[@class = 'shelf-item__title'][1]"), "Pixel 2"));
```

Comment:
css: p.shelf-item__title:first-of-type
you can do getText for this element and match it with Pixel 2